neomodel/core.py

```diff
@@ -57,6 +57,11 @@ def drop_indexes(quiet=True, stdout=None):
             stdout.write(' - Dropping index on label {0} with property {1}.\n'.format(
                 match.group(1), match.group(2)))
         else:
+            # Token lookup indexes (introduced in Neo4j 4.3)
+            # They are created automatically so they should not be dropped
+            if not index[7]:
+                assert not index[8], "Token lookup indexes have neither label nor property"
```

Comment:
assert is dangerous in case you run a project with optimization:

https://docs.python.org/3/reference/simple_stmts.html#the-assert-statement
https://docs.python.org/3/using/cmdline.html#cmdoption-O

If someone runs the project with the -O flag you will never get this exception and always reproduce the bug.

Please, replace it for the value error

Reply:
Doesn't `not index[7]` still throw the same exception, as we are accessing the array out of bounds? Shouldn't it be `len(index) < 7` instead?

Reply:
As shown in https://github.com/neo4j-contrib/neomodel/issues/567#issuecomment-877568650, `index[7]` and `index[8]`, which are lists of the labels and the names of the properties, respectively, are empty for token lookup indexes.
So `index[7]` and `index[8]` are valid, `index[7][0]` and `index[8][0]` are invalid.

Reply:
replaced with `ValueError` 4c2e46a